client/package.json

```diff
@@ -32,14 +32,14 @@
     "foundation-sites": "^6.6.3",
     "html-webpack-plugin": "^4.5.0",
     "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "^4.14.1",
+    "node-sass": "^6.0.1",
     "prop-types": "~15.7",
     "react": "~16.13",
     "react-dom": "~16.13",
     "react-hot-loader": "^4.12.21",
     "react-router-dom": "5.2",
     "redbox-react": "~1.6",
-    "sass-loader": "^8.0.2",
+    "sass-loader": "^12.1.0",
     "style-loader": "^1.2.1",
```

Comment:
these version changes will also have potential breaking changes, but node-sass is also the most frustrating dependency, so if this is the working fix Im okay with it